src/templates/stickyboard-simple/src/components/base/PageBase.js

```diff
@@ -71,6 +75,7 @@ class PageBase extends React.Component {
             currentBreakpoint: 'lg',
             layout: undefined,
             blocks: undefined,
+            EmptyStickerId: uuid(),
```

Comment:
The variable name must start with lower case letter.

Reply:
Also, I think this variable is not needed.
Please use uuid() when the empty sticker inserted.